### ISSUE TITLE
display: flow-root not detected, despite being in caniuse library

### ISSUE THREAD

**Issue by edg2s** [ismay/stylelint-no-unsupported-browser-features#276]
Failing test case:
```
testRule({
  plugins: ['.'],
  ruleName,
  config: [
    true,
    {
      browsers: ['IE 8'],
    },
  ],

  reject: [
    {
      code: 'div { display: flow-root; }',
      description: 'disallow display:flow-root for IE 8',
      message: `...`,
      line: 1,
      column: 7,
    },
  ],
});
```

**Comment by ismay** [comment]
Hi @edg2s,

I suspect that this is due to point 1 from the issue template:

> As mentioned in the readme, this plugin uses [doiuse](https://github.com/anandthakker/doiuse) to detect browser support. If you're having an issue with detection, make sure it isn't a doiuse issue by testing with doiuse directly. If the issue also occurs when using doiuse directly it should be fixed in doiuse and there's no use in opening an issue for it in this repo.

Doiuse was unmaintained for a while and stagnated because of that. But it has recently been adopted by a new maintainer, see here: https://github.com/ismay/stylelint-no-unsupported-browser-features/issues/267#issuecomment-1551297513. A whole lot of refactoring has already been done, but we're still awaiting some final work. Once that's done we should be able to support usecases like these. See also: https://github.com/ismay/stylelint-no-unsupported-browser-features/pull/275#issuecomment-1559696455.

I'll close this now, as the actual detection is doiuse territory, and should be resolved once doiuse has been updated. However, if you suspect a bug in the integration between stylelint and doiuse that's this plugin's territory, please let me know if that's the case. Also, feel free to open this issue in the doiuse repo if you can reproduce it with doiuse.
